consumer/src/test/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/oppgavebehandling/RestOppgaveBehandlingServiceImplTest.kt

```diff
@@ -394,6 +394,8 @@ class RestOppgaveBehandlingServiceImplTest {
         } returns RestOppgaveMockFactory.mockOppgaverResponse
         every { kodeverksmapperService.mapOppgavetype(any()) } returns ""
         every { RestOppgaveMockFactory.tilgangskontrollContext.checkAbac(any()) } returns AbacResponse(listOf(Response(Decision.Permit, emptyList())))
+        every { pdlOppslagService.hentIdent(any()) } returns HentIdent.Identliste(listOf(HentIdent.IdentInformasjon("07063000250", HentIdent.IdentGruppe.FOLKEREGISTERIDENT)))
```

Comment:
skulle ikke du bruke coEvery istende for Every siden vi bruker coroutines i kotlin